src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -0,0 +1,127 @@
+import { findNodeHandle } from 'react-native';
+import type GestureHandler from '../handlers/GestureHandler';
+import {
+  GestureHandlerDelegate,
+  MeasureResult,
+} from './GestureHandlerDelegate';
+import PointerEventManager from './PointerEventManager';
+import TouchEventManager from './TouchEventManager';
+import { State } from '../../State';
+import { isPointerInBounds } from '../utils';
+import EventManager from './EventManager';
+
+export class GestureHandlerWebDelegate
+  implements GestureHandlerDelegate<HTMLElement>
+{
+  private _view!: HTMLElement;
+  private gestureHandler!: GestureHandler;
+  private eventManagers: EventManager<unknown>[] = [];
+
+  get view(): HTMLElement {
+    return this._view;
+  }
```

Comment:
This may be a matter of preference, but I wouldn't mix those two conventions and I would stick to 

```suggestion
  public getView(): HTMLElement {
    return this.view;
  }
```

Reply:
Makes sense - https://github.com/software-mansion/react-native-gesture-handler/pull/2579/commits/48e70b7148bf2bd53cd954592d749940709f8979